### PR TITLE
Cleaned up IDE warnings

### PR DESCRIPTION
**Pull request by cmarcelk**
- moved some methods to be static
- fixed whitespace
- fixed hiding of variables
- broke long lines

### CODE REVIEW

**Review by agodwinrelay — APPROVED** [review]

File: relay/workflow.py [hunk 1]
```diff
@@ -457,15 +476,15 @@ def get_handler(self, event:dict):
 
             elif t == 'wf_prompt_event':
                 if event['type'] == 'started':
-                    h = self.type_handlers.get(('wf_api_prompt_start_event'), None)
+                    h = self.type_handlers.get('wf_api_prompt_start_event', None)
```
Comment (steveims) [inline relay/workflow.py:479]: are these prompt events correct?  I don't see wf_prompt_event in relay_erlang.  and do we set handlers for wf_api_prompt_start_event and _stop_event?

File: relay/workflow.py [hunk 2]
```diff
@@ -1413,97 +1341,68 @@ async def _set_device_info(self, target, field, value):
     #     await self.sendReceive(event)
     #     # await self._set_device_info(target, )
 
-    def led_info(self, rotations:int=None, count:int=None, duration:int=None, repeat_delay:int=None, pattern_repeats=None, colors=None):
```
Comment (steveims) [inline relay/workflow.py:1416]: was the intention to remove this method from the relay object?  or just remove the dependence on 'self'?

you could do the former with the @staticmethod decorator (https://docs.python.org/3/library/functions.html#staticmethod)

File: relay/workflow.py (hunk 1 above)
Comment (cmarcelk) [reply relay/workflow.py:479]: Yeah, I'm not seeing those events in ibot source or relay_erlang. I'll remove them.

**Review by steveims — COMMENTED** [review]
lgtm

**Pull request merged**